tests/main/interfaces-openvswitch-support/task.yaml

```diff
@@ -5,8 +5,8 @@ details: |
 
 # ubuntu-core, ubuntu-14, fedora, amazon are skipped as /run/uuidd/request file does not
 # exist. On those systems different files are being used instead.
-# arch: uses /run/run/uuidd/request, filed a bug report https://bugs.archlinux.org/task/58122
-systems: [-ubuntu-14.04-*,-ubuntu-core-*,-fedora-*, -arch-*, -amazon-*]
+# arch, opensuse: uses /run/run/uuidd/request, filed a bug report https://bugs.archlinux.org/task/58122
```

Comment:
Is this comment at the right place? I mean, the bug was against the arch bugtracker is marked as fixed.

Reply:
@sergiocazzolato can you file a bug at https://bugzilla.opensuse.org/index.cgi ? I'll update the test for Arch in a separate PR.

Reply:
Sorry, I think I was a bit unclear here - does suse has the same bug that they use "/run/run/uuidd" (i.e. the double /run/run) ?

Reply:
Yes, suse has the exact same bug. I tested this in tumbleweed.

Reply:
FYI https://github.com/snapcore/snapd/pull/6110/files#diff-913d16e6e81868475136850d59dc29bfR9